internal/cloudinit/cloudinit.go

```diff
@@ -97,20 +103,20 @@ swapon /swapfile
 echo "done setting up swap space"
 
 echo "downloading lite engine binary"
-/usr/bin/wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=10 "{{ .LiteEnginePath }}/lite-engine-{{ .Platform.OS }}-{{ .Platform.Arch }}" -O /usr/bin/lite-engine
+/usr/bin/wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=10 ` + liteEngineUsrBinPath + ` || /usr/bin/wget --retry-connrefused --tries=10 --waitretry=10 -nv --debug ` + liteEngineUsrBinPath + `
```

Comment:
do we need this '+' at the end?

Reply:
Since we are adding one string in other format is like this ` + string +` else it will give compilation error

Reply:
Ref: 
![Uploading Screenshot 2024-02-14 at 4.21.57 PM.png…]()